### ISSUE TITLE
Will the inference API ever allow greater than 50 rows?

### ISSUE THREAD

**Issue by ariestav** [gretelai/gretel-blueprints#363]
Hello!

In the file `docs/inference_api_beta/js/example.js` the header comment read:

```
/*
REQUIREMENTS
...
3. Maximum of 50 rows. Use the standard /models batch API for more.
...
```
Is there any intention to increase the limit, or will 50 rows always be the greatest amount of rows possible with the streaming API endpoint at `v1/inference/tabular/stream`?

Is it possible to stream results from the `/models` batch API as the suggestion states?  Or is that a situation where we must wait for the record to be created and then retrieve them.

Thank you!

**Comment by benmccown** [comment]
Hi Arie. In our Python SDK (available for pip install via PyPi) we expose a control mechanism to set the sample buffer size. See here: https://github.com/gretelai/gretel-python-client/blob/main/src/gretel_client/inference_api/tabular.py#L436

Here's our docs page covering the Python SDK. https://docs.gretel.ai/gretel-basics/getting-started/environment-setup/cli-and-sdk

We agree feature parity here makes sense and we will be updating the JS SDK, but I do not have a current time estimate to execute that work. Would utilizing our Python SDK be feasible for you in the interim? I will keep you posted via this GitHub issue as I have any updates.

**Comment by ariestav** [comment]
@benmccown Thank you for the pointers.  Unfortunately, for the environment that we are building our app for, Javascript is required.  I would totally get on the Python bandwagon, but we are supporting a run time that requires Javascript.  We definitely want to make use of it.  

I might be able to implement a work around whereby after the 50 are retrieved, the code retrieves another 50.  I might be able to send the first 50 records as "sample data" with the subsequent calls to ensure the structure of the response is consistent.

If you can offer any insight into a timeline for that singular parameter to be implemented, that would help us decide whether or not to implement the approach I'm suggesting.  Let me know your thoughts.  Thank you!

**Comment by thesteady** [comment]
Hi there! I just merged #388 which allows you to ask for more than 50 rows of data! Hopefully it's still useful to you.

**Comment by ariestav** [comment]
@thesteady Many thanks for this add!  It will definitely help.  Much appreciated!
